src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -60,16 +61,19 @@
  * SortedMap, SortedSet, String, StringBuffer, StringBuilder, SuppressWarnings, Throwable,
  * TreeMap, TreeSet, UnsupportedOperationException, Void, boolean, byte, char, double,
  * float, int, long, short, void}.
+ * Type - {@code stringSet}
```

Comment:
I think it is time to make clarification here. It is `stringArray`, and I am ready to change it everywhere in our codebase.

Reply:
Done